src/main/java/edu/ucdavis/dss/ipa/entities/SupportAssignment.java

```diff
@@ -97,4 +98,24 @@ public long getSectionGroupIdentification() {
             return 0;
         }
     }
+
+    @Transient
+    @JsonIgnore
+    public boolean isOpenToReview() {
+        // A sectionGroup required for persistence in database, but we should avoid errors if called on a DTO.
+        if (this.getSectionGroup() == null) {
+            return false;
```

Comment:
Throw an exception here. isOpenToReview() returning false is a valid condition that shouldn't be implied by bad parameters.